my-app/src/shared/layout/Sidebar.tsx

```diff
@@ -14,27 +14,27 @@ export const Sidebar = () => {
         <Flex
             flexDirection="column"
             height="100%"
-            backgroundColor="#1B004C"
+            bg="darkPrimary"
```

Comment:
Ako se sjecas Sidebar smo rekli da bi slozili kao:
```
/SidebarNavigation
   /layouts
     SidebarNavigation.mobile.tsx
     SidebarNavigation.desktop.tsx
SidebarNavigation.tsx
```

I onda bi svaki layout renderao svoju komponentu, primjer: https://infinum.com/handbook/frontend/react/project-structure#different-component-layouts-for-different-screen-sizes-in-separate-codelayoutcode-components